### PR TITLE
extended example with more variety

### PR DESCRIPTION
**Pull request by PanderMusubi**
As many people start off by changing the example provided, having more variety in the example code and also an extended implementation of view, edit and delete will help people get up to speed more quickly.

### CODE REVIEW

**Comment by PanderMusubi** [comment]
Questions for `db.Boolean`: Is `nullable=False` working properly? Are `default=True` and `default=False` supported?

**Comment by greyli** [comment]
Yes, use `default=False` will be better.

File: examples/app.py [hunk 1]
```diff
@@ -132,40 +132,58 @@ def test_flash():
...
         db.session.add(m)
     db.session.commit()
+
+@app.route('/table')
+def test_table():
     page = request.args.get('page', 1, type=int)
     pagination = Message.query.paginate(page, per_page=10)
     messages = pagination.items
-    titles = [('id', '#'), ('text', 'Message'), ('author', 'Author'), ('category', 'Category'), ('sender', 'Sender'), ('create_time', 'Create Time')]
+    titles = [('id', '#'), ('text', 'Message'), ('author', 'Author'), ('category', 'Category'), ('draft', 'draft'), ('create_time', 'Create Time')]
```
Comment (greyli) [inline examples/app.py:157]: The draft's title should be `Draft`.

**Comment by PanderMusubi** [comment]
Added your corrections.

**Pull request merged**